src/account-type-utils.js

```diff
@@ -60,7 +60,7 @@ const getAccountType = email => {
  *
  */
 const getEmailDomain = email => {
-  return email.includes('@') && email.includes('.')
+  return _.includes(email, '@') && _.includes(email, '.')
```

Comment:
Given that the cause of this seems to stem from a non-accepted ToS:
1. I would like to take a look at `fetchOk` and make sure this is not masking the issue. 
2. Ideally, I would like to provide a better error message that allows us to easily find problems like these.

How would you feel about adding a comment in the code (perhaps at `fetchOk` linking to this ticket and highlighting that we need to do more investigation?

Reply:
For context, as Pete and I were discussing on slack:
1. We don't know that the cause is non-accepted ToS for certain. It could be.
2. The fetchOk is doing a "catch-all" case where `if (!res.ok)` check, but the logs do not show any error that came from that. This makes me assume that the response from `Firecloud Orchestration` and `Thurloe` is coming up; it just comes up as `undefined` instead of the actual values.
3. Once it reaches `Bard`, the error may be washed away in either `Thurloe` or `Firecloud Orchestration`.